packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -693,7 +695,7 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
            role="combobox"
            placeholder=${placeholder}
            autocomplete="off"
-           .ariaLabel=${this.ariaLabel}
+           aria-label=${this.ariaLabel || ''}
```

Comment:
Observation in regard to accessibility, AI will raise an issue because the `aria-label` cannot be empty. However, Firefox's Inspect Accessibility Properties doesn't raise it. So, do we _ignore_ this because even though we have a _bias_ for AI?

Reply:
Just tested with AI and got no violations listed:
<img width="776" alt="image" src="https://user-images.githubusercontent.com/7122716/204591304-bce72015-2926-4b59-9964-3ec66ca67619.png">